jboss/container/wildfly/s2i/legacy/artifacts/opt/jboss/container/wildfly/s2i/assemble.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/sh
+
+source "${JBOSS_CONTAINER_UTIL_LOGGING_MODULE}/logging.sh"
+source "${JBOSS_CONTAINER_MAVEN_S2I_MODULE}/maven-s2i"
+
+if [ -n "${GALLEON_PROVISION_LAYERS}" ]; then
+   if [ -z "${GALLEON_PROVISION_FEATURE_PACKS}" ]; then
+     log_error "GALLEON_PROVISION_FEATURE_PACKS env variable must be set when GALLEON_PROVISION_LAYERS is set"
+     exit 1
+   fi
+fi
+if [ -n "${GALLEON_PROVISION_FEATURE_PACKS}" ]; then
+   if [ -z "${GALLEON_PROVISION_LAYERS}" ]; then
+     log_error "GALLEON_PROVISION_LAYERS env variable must be set when GALLEON_PROVISION_FEATURE_PACKS is set"
+     exit 1
+   fi
+fi
+if [ -n "${GALLEON_PROVISION_FEATURE_PACKS}" ]; then
```

Comment:
could we add a message that these 2 env vars are deprecated and should be replaced by the usage of the plugin?
So we can have the opportunity to actually remove them in later versions at some point

Reply:
+1